core/Cargo.toml

```diff
@@ -52,7 +52,7 @@ urlencoding = "2.1"
 url = "2.5"
 dns-lookup = "1.0"
 async-stream = "0.3"
-eventsource-client = { git = "https://github.com/dust-tt/rust-eventsource-client", rev = "d34896e77b2198cd4d71e05007798736142ec336" }
+eventsource-client = { git = "https://github.com/dust-tt/rust-eventsource-client", rev = "148050f8fb9f8abb25ca171aa68ea817277ca4f6" }
```

Comment:
Gonna have to fight the LSP again :p

Reply:
This time for good!

Reply:
(moved from commit to rev which is what we wanted all along)